src/app/company-detail/company-detail.component.ts

```diff
@@ -16,35 +17,37 @@ import { SingleExperienceComponent } from '../single-experience/single-experienc
 })
 export class CompanyDetailComponent {
   dataService: DataService = inject(DataService);
-  selectedCompany: Company = {
-    backgroundImageUrl: 'assets/background/meta_cover.jpeg',
-    companyDetailLogo: 'assets/company-logos/meta_logo.jpeg',
-    companyReviews: 2456,
-    companySize: 86000,
-    competitors: 'Snap, Youtube, Twitter',
-    description:
-      'Meta builds technologies that help people connect, find communities, and grow businesses. When Facebook launched in 2004, it changed the way people connect. Apps like Messenger, Instagram and WhatsApp further empowered billions around the world. Now, Meta is moving beyond 2D screens toward immersive experiences like augmented and virtual reality to help build the next evolution in social technology.',
-    founded: '2004',
-    headquarters: 'Menlo Park, CA',
-    id: 4,
-    industry: 'Internet & Web Services',
-    isBookmarked: false,
-    logoImageUrl: 'assets/facebook.png',
-    name: 'Meta',
-    rating: 4,
-  };
+  
+  // arrays of Company objects
   companiesList: Company[] = [];
   companyJobs: Job[] = [];
   suggestedCompanies: Company[] = [];
+  // specificCompany of Company object
+  specificCompany: Company | undefined;
+
+  constructor(private route: ActivatedRoute) { }
+
+  ngOnInit(): void {
+    let companyId: number = 0; // initialize default value
```

Comment:
@AleenaTim @Ishiano change this to `const` instead of `let` since you are not reassigning a new value to it later in the code.